classes/HandlePreflight.php

```diff
@@ -33,7 +34,10 @@ public function handle($request, Closure $next)
             $preflight = $this->cors->handlePreflightRequest($request);
             $response->headers->add($preflight->headers->all());
         }
-        $response->setStatusCode(204);
+
+        $responseStatusCode = Settings::instance()->response_status_code ?? 204;
```

Comment:
I'd prefer such syntax but please make sure that it does work in the same way:

```suggestion
        $responseStatusCode = Settings::get("response_status_code", 204);
```